cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/server/CmisEventDispatcher.java

```diff
@@ -100,7 +100,7 @@ else if (XmlUtils.isWellFormed(result, "cmis")) {
 		catch (ListenerException e) {
 			if(e.getCause() instanceof PipeRunException) {
 				PipeRunException pre = (PipeRunException) e.getCause();
-				if(pre.getCause() != null && pre.getCause() instanceof CmisBaseException)
+				if(pre != null && pre.getCause() instanceof CmisBaseException)
```

Comment:
Hier moet dan toch ook een pre.getCause() != null bij

Reply:
Nee, toch? Als die null is dan is `instanceof` false

Reply:
Google says...